com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariable.cs

```diff
@@ -96,6 +103,20 @@ public virtual T Value
             }
         }
 
+        // Compares two values of the same unmanaged type bitwise
+        // Ignoring any overriden value checks
+        // Size is fixed
+        private static unsafe bool ValueEquals(ref T a, ref T b)
```

Comment:
```suggestion
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        private static unsafe bool ValueEquals(ref T a, ref T b)
```

the comment block is out of date.
also what about hinting compiler to inline it too?

Reply:
How is the comment out of date?

Could be inlined but I think it's premature optimization.

Reply:
I mean I think its fair that we should at least tell the compiler it could inline if possible. I mean inlining is a common practice where it makes sense would you not agree? I mean marking something as inline in say C++ is done with a purpose and I would expect no difference here?